### PR TITLE
Add 'How to install' section and requirements.txt

### PR DESCRIPTION
**Pull request by wikijm**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+asn1crypto==0.24.0
+beautifulsoup4==4.6.0
+bs4==0.0.1
```
Comment (cclauss) [inline requirements.txt]: Is this needed given that __beautifulsoup4__ is just above?

**Comment by wikijm** [comment]
Hi @cclauss,

Honestly, my python skills are limited 😞 

As I can say is:
- I'm not sure about asn1crypto usage. Feel free to remove it
- When I uninstall beautifulsoup4 and bs4, I get this error:
```
Traceback (most recent call last):
  File "SI_login.py", line 16, in <module>
    from bs4 import BeautifulSoup
ImportError: No module named bs4
[Error] Unable to login to LinkedIn.com
```

**Comment by cclauss** [comment]
Keep __beautifulsoup4__ in and remove __bs4__.

**Comment by wikijm** [comment]
Done! 😃

**Pull request merged**